### PR TITLE
bug/2326-simple-passwords: re-enabling simple passwords restriction for prod

### PR DESCRIPTION
**Pull request by patrickhuie19**
**This is a breaking change**

Insecure configs are no longer allowed on production builds.

### CODE REVIEW

**Comment by essamhassan** [comment]
PRs that make me happy :)
did we confirm no prod use in the wild?

**Comment by patrickhuie19** [comment]
> PRs that make me happy :) did we confirm no prod use in the wild?

We feel confident enough in our communication + heads up time to move forward with this.

**Review by essamhassan — APPROVED** [review]
LGTM, thanks Patrick

File: integration-tests/utils/templates/node_secrets.go [hunk 1]
```diff
@@ -9,11 +9,10 @@ import (
 
 var NodeSecretsTemplate = `
 [Database]
-URL = 'postgresql://postgres:test@{{ .PgHost }}:{{ .PgPort }}/{{ .PgDbName }}?sslmode=disable' # Required
-AllowSimplePasswords = true
+URL = 'postgresql://postgres:mysecretpassword@{{ .PgHost }}:{{ .PgPort }}/{{ .PgDbName }}?sslmode=disable' # Required
```
Comment (patrickhuie19) [inline integration-tests/utils/templates/node_secrets.go:12]: Can we store the db password here as secret as well? i.e. something like 
`postgresql://postgres:{{ .PgPass }}@{{ .PgHost }}:{{ .PgPort }}/{{ .PgDbName }}?sslmode=disable`

Comment (skudasov) [reply integration-tests/utils/templates/node_secrets.go:12]: This code will be replaced in https://github.com/smartcontractkit/chainlink/pull/10067, but sure.

**Review by skudasov — APPROVED** [review]

**Pull request merged**